selfservice/strategy/oidc/.snapshots/TestSettingsStrategy-suite=unlink-case=should_not_be_able_to_unlink_an_non-existing_connection-flow=fetch.json

```diff
@@ -111,8 +111,8 @@
         "context": {
           "provider": "github"
         },
-        "id": 1050002,
-        "text": "Link github",
+        "id": 1050003,
+        "text": "Unlink github",
```

Comment:
Thats because I changed the `agent` for this specific test from `"oryer"` to `"githuber"`(https://github.com/ory/kratos/pull/3185/files#diff-4e529852fafcb4df947a8b3f3d61bf48a522ca87c1a926bc182eb36efb051bc9L287-R286)